ide/rust/RepoCargoManifestGenerator.kt

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
Given a Bazel repository (either the current project or an external dependency such as `@vaticle_typeql`, `RepoCargoManifestGenerator.generateManifests()` will run the **Rust IDE Sync Info Aspect**, generating IDE metadata files in `bazel-bin`, and then scan the generated files and use them to generate `Cargo.toml` files that constitute a Cargo project.